src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -17,6 +17,8 @@ A couple of hooks were modified between 1.7.0.x and 1.7.1.x.
 * `displayProductButtons` has been renamed into `displayProductAdditionalInfo`.<br>
   Don’t worry, we kept an alias :)
 
+Some hooks have been modified in the order view page in version {{< minver v="1.7.7" >}} see [Modified order hooks][modified-order-hooks] for more details.
```

Comment:
I think this whole block can be removed, actually. All the detailed information should be noted in the "core updates" section.

Reply:
They are in the core updates page, I actually wanted to use this link in related hooks so that users can find more details easily But since the whole hook display relies on a `{{% funcdef %}}` I can't integrate links to the hook description.. So that's the best I found to keep a link available in this page
I'm not sure everyone knows about these core update pages, especially when they are just looking for info, but if you think they are enough I'll remove this block

Reply:
Oh of course you can add links within a `funcdef` block. You just can't put references outside it:

```
{{% someshortcode %}}
[This link works]({{< ref "something.md" >}})
{{% /someshortcode %}}

{{% someshortcode %}}
[This link doesn't][some-link]
{{% /someshortcode %}}

[some-link]{{< ref "something.md" >}}
``